GITFLOW.md

```diff
@@ -136,7 +134,6 @@ Don't forget to update the Changelog and the version in `setup.py`.
 
 - Open a Pull Request against `master`
```

Comment:
First you need to checkout against the specific release tag you want to do the fix `git checkout master@2.5` for example and create a branch names hotfix/** . Then do your fixes and push the branch. Then create a release tag on this branch